docker-compose.yml

```
@@ -1,63 +1,67 @@
 version: '3'
+
+x-env-generic-backend: &x-env-generic-backend
+  TZ: Europe/Stockholm
+  WHARF_API_URL: http://api:8080
+  ALLOW_CORS: "YES"
+
 services:
   proxy:
     image: amd64/traefik:v2.5
     ports:
     - "5000:8081"
     volumes:
     - ./wharf-docker-compose/traefik:/etc/traefik # Traefik configuration file
-  web:
-    image: quay.io/iver-wharf/wharf-web
+#  web:
+#    image: quay.io/iver-wharf/wharf-web
   api:
     image: quay.io/iver-wharf/wharf-api
     ports:
     - "5001:8080"
     environment:
-    - DBHOST=db
-    - DBUSER=postgres
-    - DBPASS=OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
-    - DBPORT=5432
-    - DBNAME=wharf
-    - DBLOG=true
-    - RABBITMQUSER=guest
-    - RABBITMQPASS=guest
-    - RABBITMQHOST=rabbitmq
-    - RABBITMQPORT=5672
-    - RABBITMQVHOST=
-    - RABBITMQNAME=wharf_queue
-    - RABBITMQDISABLESSL=TRUE
-    - RABBITMQCONNATTEMPTS=10
-    - MOCK_LOCAL_CI_RESPONSE=true
-    - ALLOW_CORS=YES
-    - WHARF_INSTANCE=local
+      <<: *x-env-generic-backend
+      DBHOST: db
+      DBUSER: postgres
+      DBPASS: OL2AEn6lgj6ekajgKJIOanefgegnksngpoetPIEQjhankf7412
+      DBPORT: 5432
+      DBNAME: wharf
+      DBLOG: "true"
+      RABBITMQUSER: guest
+      RABBITMQPASS: guest
+      RABBITMQHOST: rabbitmq
+      RABBITMQPORT: 5672
+      RABBITMQVHOST:
+      RABBITMQNAME: wharf_queue
+      RABBITMQDISABLESSL: "TRUE"
```

nit: While you have tampered with these, could you make this "TRUE" lowercase for consistency? I am sure that the case does not matter when it is parsed.

```suggestion
      RABBITMQDISABLESSL: "true"
```